packages/agents/src/react.tsx

```diff
@@ -134,7 +195,7 @@ export function useAgent<State = unknown>(
   }) as PartySocket & {
     agent: string;
     name: string;
-    setState: (state: State) => void;
+    setState: (state: unknown) => void;
```

Comment:
why change this signature?

Reply:
Hmm, my first approach was without the State parameter, and I didn't revert those changes when I added it to the new overload function.

I'll revert those changes.